spec/cypress/app_commands/scenarios/basic.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# You can setup your Rails state here
+# MyModel.create name: 'something'
+Post.create!(title: "I am a Postman")
```

Comment:
Is this needed?

Reply:
no, but I wanted to keep it so that people know scenarios exist instead of removing the directory completely - it can be deleted when someone adds an actual scenario